index.html

```diff
@@ -7,13 +7,98 @@
   <title>My Resume</title>
```

Comment:
Make sure you add a meta viewport inside your head so that your responsive units are based off the viewport!